cli.js

```diff
@@ -1,6 +1,19 @@
 #!/usr/bin/env node
 var fs = require('fs')
-var args = require('minimist')(process.argv.slice(2), {boolean: ['prune', 'asar', 'all', 'overwrite', 'strict-ssl']})
+var args = require('minimist')(process.argv.slice(2), {
+  boolean: [
+    'prune',
+    'asar',
+    'all',
+    'overwrite',
+    'strict-ssl',
+    'download.strictSSL'
+  ],
+  default: {
+    'strict-ssl': true,
```

Comment:
Yeah, wow. Good catch. This means that users using electron-packager v6 from the command line will accept invalid ssl certificates.

This is bad enough that I would recommend running `npm deprecate` on all older versions so users know to upgrade.